.github/workflows/markdown-link-check.yml

```diff
@@ -0,0 +1,17 @@
+# Checks for any broken links in markdown files
+name: Check Markdown links
+
+on: workflow_call
+
+jobs:
+  check-links:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: gaurav-nelson/github-action-markdown-link-check@v1
+      with:
+        use-verbose-mode: 'yes'
+        config-file: '.github/workflows/markdown-link-check.config.json'
+        folder-path: './examples'
```

Comment:
Do we want to limit to just the `folder-path` or we let the plugin to check all markdown files.

Reply:
I think it's okay to check all markdown files. I feel like it'll be a lot more work to add every folder path for every example. We also want to check _every_ link that is public to make sure that it isn't broken.

Reply:
If that's what we want, then shall we remove this line `folder-path` as reading here, that is not default behaviour unless `folder-path` is defined.

Reply:
Sounds good. Removed `file-path` too so that we check every single `.md` file.